bin/workspace_root.ml

```diff
@@ -40,8 +40,10 @@ module Candidate = struct
     }
 end
 
-let find () =
-  let cwd = Sys.getcwd () in
+let find from =
+  let cwd =
+    if Filename.is_relative from then Filename.concat (Sys.getcwd ()) from else from
```

Comment:
How about `Fpath.initial_cwd`? The old code didn't use it before, but at least it only called `getcwd` once.

Reply:
Good idea, done.